documents/motions/11.2019.1 - Finances.md

```diff
@@ -42,3 +43,7 @@
       6. fees charged by the WCA for activities done by the WCA or WCA Staff for other parties
       7. gifts, grants, and inheritances
       8. any other income approved by the WCA Board of Directors
+10. Filing
+    1. The WCA Financial Committee shall deliver to the WCA Board of Directors the annual returns to be filed, both on a state and federal level, at least 30 days prior to the date they are due to be filed.
+    2. Upon approval of the WCA Board of Directors, the WCA Financial Committee shall file any necessary returns on behalf of the WCA.
+    3. The WCA Financial Committee shall publish any returns open to public inspection and an annual financial report on the WCA website.
```

Comment:
```suggestion
    3. The WCA Financial Committee shall publish all returns open to public inspection and an annual financial report on the WCA website.
```

Reply:
SGTM!

Reply:
+1

Reply:
+1

Reply:
I have already included this in a `Staff review` commit that I'll be pushing live later.